apps/prairielearn/elements/pl-number-input/pl-number-input.py

```diff
@@ -75,10 +77,20 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
     name = pl.get_string_attrib(element, "answers-name")
     pl.check_answers_names(data, name)
 
-    correct_answer = pl.get_float_attrib(element, "correct-answer", None)
+    correct_answer = pl.get_string_attrib(element, "correct-answer", None)
```

Comment:
Is the purpose here to check if the correct answer is present as an attribute? There is a pl.has_attrib function that serves this purpose.